src/program/lwaftr/run/run.lua

```diff
@@ -42,6 +42,12 @@ local function migrate_device_on_config(config, v4, v6)
          queue.external_interface.device = v6
       end
    end
+
```

Comment:
WDYT about doing this at line 150 or so:

```lua
if opts['on-a-stick'] then assert(not v6); v6 = v4 end
```

then rely on the existing code doing the right thing.  WDYT?